rclcpp_components/test/test_component_manager_api.cpp

```diff
@@ -211,6 +211,50 @@ TEST_F(TestComponentManager, components_api)
     EXPECT_EQ(result->unique_id, 0u);
   }
 
+  {
+    // use_global_arguments
+    auto request = std::make_shared<composition_interfaces::srv::LoadNode::Request>();
+    request->package_name = "rclcpp_components";
+    request->plugin_name = "test_rclcpp_components::TestComponentFoo";
+    request->node_name = "test_component_global_arguments";
+    rclcpp::Parameter use_global_arguments("use_global_arguments",
+      rclcpp::ParameterValue(true));
+    request->extra_arguments.push_back(use_global_arguments.to_parameter_msg());
+
+    auto future = composition_client->async_send_request(request);
+    auto ret = exec->spin_until_future_complete(future, 5s);  // Wait for the result.
+    auto result = future.get();
+    EXPECT_EQ(ret, rclcpp::FutureReturnCode::SUCCESS);
+    EXPECT_EQ(result->success, true);
+    EXPECT_EQ(result->error_message, "");
+    std::cout << result->full_node_name << std::endl;
+    EXPECT_EQ(result->full_node_name, "/test_component_global_arguments");
+    EXPECT_EQ(result->unique_id, 7u);
```

Comment:
@gezp nit: this tests that the manager did not fail but not that it did anything about `use_global_arguments`.

Reply:
yes, `use_global_arguments`  is set true in `NodeOption` , but this component didn't use any global arguments, i don't know how to test for this.